bin/format.sh

```diff
@@ -14,6 +14,4 @@ java --add-exports jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED \
 retVal=$?
 if [ $retVal -ne 0 ]; then
     echo "[ERROR]: Some files needed formatting"
-    exit $retVal
```

Comment:
Is this necessary for the Github Action? Otherwise, a non-zero return value may be useful when the format script is used as (pre-commit) git hook to prevent committing code that does not satisfy the format rules.

Reply:
Re added in https://github.com/cbl/algorithm-analyzer/pull/4